<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphWindow</class>
 <widget class="QDialog" name="graphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>474</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Graph Window</string>
  </property>
  <widget class="QLineEdit" name="destinationInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; /* Adjust the value as needed */
background-color : white;
color: black;</string>
   </property>
   <property name="placeholderText">
    <string>Enter destination document</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findPathButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 1px solid white;
    background-color: transparent;
	border-radius: 20px; /* Adjust the value as needed */
}

QPushButton:hover {
    color: rgb(143, 0, 156);
    border: 1px solid transparent;
    background-color: white;
}</string>
   </property>
   <property name="text">
    <string>Find Path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; /* Adjust the value as needed */
background-color : white;
color: black;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter source document</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {
background-color: rgba(255, 255, 255, 30);
    line-height: 200%; /* Adjust the percentage as needed */
	border: 1px solid white;
	border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>------</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>31</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white
</string>
   </property>
   <property name="text">
    <string>Pathfinder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
     <underline>true</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Graph (adjacency list):</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>1001</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../purple.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>destinationInput</zorder>
  <zorder>findPathButton</zorder>
  <zorder>sourceInput</zorder>
  <zorder>textBrowser</zorder>
  <zorder>pathLabel</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
